backend/cfg/cfg_to_linear_desc.ml

```diff
@@ -52,5 +52,6 @@ let from_basic (basic : basic) : Linear.instruction_desc =
         Iname_for_debugger
           { ident; which_parameter; provenance; is_assignment; regs }
       | Dls_get -> Idls_get
+      | Poll -> Ipoll { return_label = None }
```

Comment:
`Linearize` has [an optimization for back-edges](https://github.com/ocaml-flambda/flambda-backend/blob/74131893886b362d8a7676e4ba49905afae41238/backend/linearize.ml#L191-L195) for this case. I am not sure if we are getting it on `main` with `Cfg`, depends on how good our `cfgize` and fallthrough elimination passes are, but this PR is not going to have it. It may be  hard to fix as part of this PR, but it would be nice to at least have a test and open an issue to track this.

Reply:
(I have created https://github.com/ocaml-flambda/flambda-backend/issues/2228 to track it.)